CHANGELOG.md

```diff
@@ -50,6 +50,9 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 Here is an overview of all **stable** additions:
 
+
+
+- **General:** Introduce new Neo4j Scaler ([#3999](https://github.com/kedacore/keda/issues/3999))
```

Comment:
Could you please sort this alphabetically, ie put it after following line:
```
- **General**: Introduce new ArangoDB Scaler ([#4000](https://github.com/kedacore/keda/issues/4000))
```

Reply:
Sure